public/static/locales/cy/landing.json

```diff
@@ -37,31 +37,31 @@
-          "answer": "<article>Os yw unrhyw un o'r canlynol yn berthnasol yna mae'n werth cysylltu â chwmni arbenigol i weld a allant gynnig bargen well neu ratach: <ul style='margin-left: 30px; list-style-type: disc;'><li>You have been refused travel insurance because of a medical condition.</li> <li>An existing policy has been cancelled or amended because of a medical condition.</li><li>You have been offered insurance, but your medical condition has been excluded.</li><li>You have been offered insurance with a high additional premium because of your medical condition.</li></ul>The firms on our Directory specialise in providing travel insurance to people who have serious and/or multiple medical conditions that carry a high risk of needing some form of medical intervention while you are on your trip.  Examples of conditions might be cancer, stroke or serious heart condition, respiratory conditions or conditions that have been diagnosed as terminal.<br/>However, different firms have different views on what conditions are ‘high risk’ and its worth trying more than one.</article>"
+          "answer": "<article class='accordion'><p>Os yw unrhyw un o'r canlynol yn berthnasol yna mae'n werth cysylltu â chwmni arbenigol i weld a allant gynnig bargen well neu ratach:</p> <ul><li>You have been refused travel insurance because of a medical condition.</li> <li>An existing policy has been cancelled or amended because of a medical condition.</li><li>You have been offered insurance, but your medical condition has been excluded.</li><li>You have been offered insurance with a high additional premium because of your medical condition.</li></ul><p>The firms on our Directory specialise in providing travel insurance to people who have serious and/or multiple medical conditions that carry a high risk of needing some form of medical intervention while you are on your trip.  Examples of conditions might be cancer, stroke or serious heart condition, respiratory conditions or conditions that have been diagnosed as terminal.</p><p>However, different firms have different views on what conditions are ‘high risk’ and its worth trying more than one.</p></article>"
         },
         {
           "question": "I haven’t been offered any insurance yet – I don’t know if I need a specialist.",
-          "answer": "<article>Then it’s worth trying a mainstream provider AND a couple of specialists to compare prices and cover offered. You can do this yourself – or you can ask an Insurance Broker to help you. The box at the bottom of this screen <strong>Need help shopping around?</strong> will tell you how.</article>"
+          "answer": "<article class='accordion'><p>Then it’s worth trying a mainstream provider AND a couple of specialists to compare prices and cover offered. You can do this yourself – or you can ask an Insurance Broker to help you. The box at the bottom of this screen <strong>Need help shopping around?</strong> will tell you how.</p></article>"
```

Comment:
There are a number of pieces of copy that do not have translations.

Reply:
Yeah they havent been supplied yet but i can use google to add something there for now.

Reply:
OK, no problem. Better to get the correct copy than add a temporary one though so for the purpose of this PR I'd say to leave that as it is.